sf-docker/run-jetson.sh

```diff
@@ -57,6 +57,11 @@ sed -i "s/S3_ENDPOINT=.*/S3_ENDPOINT=http:\/\/$(hostname):9000/g" .env.sfstation
 echo $VERSION
 echo $REGISTRY
 
+# create mqtt user for rmq mqtt plugin
+docker exec -it rmq /opt/rabbitmq/sbin/rabbitmqctl add_user mqtt mqtt || true
```

Comment:
isn't possible to move those commands to sf_dependencies compose entrypoint?

Reply:
not easily, as these commands need to run after RMQ fully starts. That means the entrypoint script would need to start RMQ server process in background, then either sleep for some time or wait for the RMQ to start in some other way, then create the user and then bring back the RMQ server process to foreground

on the other hand the `run.sh` script already prepares other dependencies such as creating SQL database, S3 bucket and others, so it was way easier to add it there.

IMO if we will ever try to remove the run script, we can also add this logic to the entrypoint

Reply:
ok